charts/prometheus-node-exporter/templates/daemonset.yaml

```
@@ -136,9 +136,15 @@ spec:
           volumeMounts:
             - name: proc
               mountPath: /host/proc
+              {{- if .Values.hostProcFsMount.enabled }}
+              mountPropagation: {{ .Values.hostProcFsMount.mountPropagation }}
+              {{- end }}
               readOnly:  true
             - name: sys
               mountPath: /host/sys
+              {{- if .Values.hostSysFsMount.enabled }}
+              mountPropagation: {{ .Values.hostSysFsMount.mountPropagation }}
+              {{- end }}
```

See the comment above.

```suggestion
              {{- with .Values.hostSysFsMount.mountPropagation }}
              mountPropagation: {{ . }}
              {{- end }}
```

---

Change made.